doc/nrf/external_comp/dult.rst

```diff
@@ -12,7 +12,12 @@ Following the specification improves the privacy and safety of individuals by pr
 For detailed information about supported functionalities, see the official `DULT`_ documentation.
 
 .. note::
+   The `DULT`_ documentation has the Internet-Draft status.
+   Internet-Drafts are draft documents valid for a maximum of six months and may be updated, replaced, or obsoleted by other documents at any time.
```

Comment:
Consider explicitly marking it as a quite from `DULT status`

Reply:
@annwoj, how can we mark it as a quote?

Reply:
oh, nvm, @annwoj  suggested to rephrase it entirely